packages/workflows/src/definitions.ts

```diff
@@ -11,4 +12,13 @@ export enum InputAlias {
 
   AttachedInventoryItems = "attachedInventoryItems",
   DetachedInventoryItems = "detachedInventoryItems",
+
+  Cart = "cart",
+
+  LineItems = "lineItems",
+  ShippingOptionToValidate = "shippingOptionToValidate",
```

Comment:
Suggestion: I think it is a bit to specific here, shipping options wouldn't be enough?